### PR TITLE
Fixed all accessibility colors. Again.

### PR DESCRIPTION
**Pull request by MaanavD**
Should address the last 300+ accessibility color issues.

### CODE REVIEW

**Review by natke — APPROVED** [review]

**Review by natke — CHANGES_REQUESTED** [review]
Looks like Ruby can't be installed for checklinks?

**Comment by MaanavD** [comment]
@natke Fixed checklinks after blood sweat and tears. Also, linted many files. Also, fixed HTTPS links. Please review.

image

**Comment by MaanavD** [comment]
all changes viewable here: https://maanavd.github.io/onnxruntime/

File: .github/workflows/check-website-links.yml [hunk 1]
```diff
@@ -37,7 +39,28 @@ jobs:
...
+          bundle exec htmlproofer ./_site \
+            --only_4xx \
+            --ignore-status-codes 429,403 \
+            --allow_hash_href \
+            --allow_missing_href \
+            --ignore_urls "/.*linkedin\.com.*/,https://onnxruntime.ai/docs/reference/api/c-api.html,https://www.onnxruntime.ai/docs/reference/execution-providers/TensorRT-ExecutionProvider.html#c-api-example,https://www.onnxruntime.ai/docs/resources/graph-optimizations.html,onnxruntime/capi/onnxruntime_pybind11_state.html,https://github.com/microsoft/onnx-converters-private/issues/new/choose,https://aka.ms/onnx/exportissue,https://aka.ms/onnx/board" \
+            --no-check-external-hash
```
Comment (natke) [inline .github/workflows/check-website-links.yml:50]: I assume this didn't work?

**Review by natke — APPROVED** [review]

**Pull request merged**